CIP-XXXX/README.md

```diff
@@ -0,0 +1,447 @@
+            },
+            "6": "Virtua Metaverse"
+          }
+        },
+        "27": {
+          "0": "0.05",
+          "1": [
+            "addr_test1qqp7uedmne8vjzue66hknx87jspg56qhkm4gp6ahyw7kaahevmtcux",
+            "lpy25nqhaljc70094vfu8q4knqyv6668cvwhsq64gt89"
+          ]
+        }
+      }
+    },
+    "2": [
+      [
+        "e97316c52c85eab276fd40feacf78bc5eff74e225e744567140070c3j",
+        [
+          "witness may be broken up into a sub-array of 64-character strings",
+          "assuming that the overall length of the witness is longer than the",
+          "64-character on-chain metadata limit for strings."
+        ]
+      ],
+      [
+        "26bacc7b88e2b40701387c521cd0c50d5c0cfa4c6c6d7f0901395757",
+        [
+          "this is the second witness for a 2 out of 3 multi-sig script.",
+          "it's also broken up into an array.",
+          "strings in the witness array should be concatenated as-is",
+          "withouts spaces or any formatting applied"
+        ]
+      ]
+    ]
+  }
+}
+```
+
+### Example FT Token Registration Metadata
+
+```json
+{
+  "867": {
+    "1": {
+      "1": "00000002df633853f6a47465c9496721d2d5b1291b8398016c0e87ae",
+      "2": [
+        26
+      ],
+      "3": [
+        "Ed25519"
+      ],
+      "4": 12345,
+      "5": "https://oracle.tokenproject.io/",
+      "6": {
+        "26": {
+          "1": [
+            {
+              "0": [
+                "d894897411707efa755a76deb66d26dfd50593f2e70863e1661e98a0",
+                "7370616365636f696e73"
+              ],
+              "1": "spacecoins",
+              "2": "SPACE",
+              "3": [
+                "the OG Cardano community token",
+                "-",
+                "whatever you do, your did it!",
+                "",
+                "Learn more at https://spacecoins.io!"
+              ],
+              "4": 0,
+              "5": [
+                "https://",
+                "spacecoins.io"
+              ],
+              "6": [
+                "ipfs://",
+                "bafkreib3e5u4am2btduu5s76rdznmqgmmrd4l6xf2vpi4vzldxe25fqapy"
+              ],
+              "7": [
+                [
+                  "ipfs://",
+                  "bafkreibva6x6dwxqksmnozg44vpixja6jlhm2w7ueydkyk4lpxbowdbqly"
+                ],
+                "3507afe1daf05498d764dce55e8ba41e4acecd5bf42606ac2b8b7dc2eb0c305e"
+              ],
+              "8": [
+                "37da68d092f4e61feb237de5e86b404171b59d3880d340023a16a6983491736d",
+                "0"
+              ]
+            }
+          ]
+        }
+      }
+    },
+    "2": [
+      [
+        "e97316c52c85eab276fd40feacf78bc5eff74e225e744567140070c3j",
+        [
+          "witness may be broken up into a sub-array of 64-character strings",
+          "assuming that the overall length of the witness is longer than the",
+          "64-character on-chain metadata limit for strings."
+        ]
+      ]
+    ]
+  }
+}
+```
+
+## Rationale
```

Comment:
It appears to me that this proposal goes some way to address [CPS-0001](https://github.com/cardano-foundation/CIPs/pull/371), might be nice to mention this CPS here. 

Also it would be good to add more to this section to articulate the potential alternative designs.

Reply:
yes: at least the first part of this suggestion was well received at today's CIP meeting (cc @ehanoc)

Reply:
Thank you, I will review that CPS and add references as appropriate.

Can you expand on what you mean by "articulate the potential alternative designs"?

Reply:
> Can you expand on what you mean by "articulate the potential alternative designs"?

What I mean is, what could be competing design elements to the ones in this CIP? if any? -- for example: is there any alternative to storing this information in transaction metadata?

Reply:
@Ryun1 there was an alternative framework that came along last year but it was not very popular & has been waiting in the wings now for a few months.  It may be useful to have a look at these as alternatives for comparison: https://github.com/cardano-foundation/CIPs/pull/467#issuecomment-1471481889

Reply:
@Ryun1 @rphair I have added reference to CPS-0001 in the latest push to update the readme document in the "Motivation" section. I have also amended the CDDL slightly after reviewing and absorbing CPS-0001 to provide flexibility in "scoping" CIP-88 in the future to support additional "scopes" beyond tokens should there be community interest in such.

Reply:
Since CPS-0001 is now covered, and the only alternative designs I was aware of (https://github.com/cardano-foundation/CIPs/pull/467#issuecomment-1471481889) appear not to be significant in the community, @Ryun1 I think this question is resolved (you can mark it if so).